### ISSUE TITLE
Hero

### ISSUE THREAD

**Issue by rosejbon** [thelawcentresnetwork/lcn-design-system#44]

**Comment by rosejbon** [comment]
There are two types of heros for general pages. One for the home page and one for the rest.

**Comment by harryharrold** [comment]
There seems to be a third header - on the News item page?

**Comment by rosejbon** [comment]
@harryharrold I was thinking that would be a part of the news page template, but happy to be challenged on that assumption.

**Comment by rosejbon** [comment]
We ran out of time to add images as an option to this component - https://github.com/thelawcentresnetwork/lcn-design-system/issues/83

And I have sharded off the Homepage hero into a separate ticket for the same reason.

**Comment by jermyk** [comment]
@rosejbon I was able to change the colour as per the AC but noticed if I typed in "White" with a capital, it changed the text  colour to black?
![Content - White-Black](https://github.com/thelawcentresnetwork/lcn-design-system/assets/4446325/9f9eb696-2fe5-4e85-8a07-a86d3ea5670b)

**Comment by rosejbon** [comment]
@jermyk this is correct, there are set colours that we can use and they are case sensitive, I will change the documentation to reflect the options
